rstudio/shinytest2
# Add a `usethis` style method; Open `record_test()` in `Chrome` or external browser

Fixes #107
Fixes #91 

Changes
* Tighten package deps
* Add `use_shinytest()`
* Open `record_test()` in `chromote::Chrome` browser or by setting `external` viewer for RStudio IDE
	* Use `chromote::default_chromote_object()$new_session()` when creating new `ChromoteSessions` for `AppDriver`.
	* Following this behavior, we can use the `default_chromote_object()$get_brower()` to see if it is a `Chrome` instance.
* `test_app()` performs an expectation if in testing mode to pass test chunks. Ex: `test_that("my test", { test_app(path_to_app) })`
* Add internal helper method for `shinycoreci-apps` migration
